pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.6
+title: Release Notes for 1.10.6
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.6.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Issues Fixed in DC/OS 1.10.6
+
+- DCOS-19865 - Fixed focus/change/blur behavior.
+- SOAK-68/DCOS-21451 - Fixed 404 error returned by adminrouter on Maration UI/Prevented the reuse of tcp sockets by AR's cache code [1.11].
+
+# Notable Changes in DC/OS 1.10.6
+
+- COPS-1879/COPS-2166/DCOS_OSS-2315 - Bumped Mesos SHA to the latest 1.4.x version. ([changelog](https://github.com/mesosphere/mesos/compare/c8bfe874c68cdf6b9d7e4b7ab0dd81c45b26bc79...178a8addd30f5f4e72c5aa3d61bc07e15e75fe87)).
```

Comment:
As we discussed, this link might be more accurate and readable for users:
https://github.com/mesosphere/mesos/blob/dcos-mesos-1.4.x-24d3886/CHANGELOG